src/main/resources/static/analytics.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+    <title>Analytics | mini-SurveyMonkey</title>
+    <link rel="icon" href="logo/icon.png">
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+
+</head>
+
+<body>
+    <div>
+        <nav class="py-2 bg-light border-bottom">
+            <div class="container d-flex flex-wrap">
+                <ul class="nav me-auto">
+                    <li class="nav-item"><a href="index.html" class="nav-link link-dark px-2">Home</a></li>
+                    <li class="nav-item"><a href="./about.html" class="nav-link link-dark px-2 active"
+                            aria-current="page">About</a></li>
+                </ul>
+                <ul class="nav">
+                    <li class="nav-item"><a href="user.html" class="nav-link link-dark px-2">User</a></li>
+                    <li class="nav-item"><a href="surveyor.html" class="nav-link link-dark px-2">Surveyor</a></li>
+
+                </ul>
+            </div>
+        </nav>
+        <h1 class='display-5 text-center mt-3 pt-3'>Survey Results</h1>
```

Comment:
Can probably display the actual survey name and put "Results" at the end

Reply:
I actually added the survey name in the last commit. forgot to show that in the screenshot